### DISCUSSION TITLE
QNAP Container Station

### DISCUSSION THREAD

**Discussion by kle1nIT** [dani-garcia/vaultwarden#3420] [General]
What i´m doing wrong ? 

I can't quite figure out the Container Station app.

I have created a shared folder in which I have created the subfolder Vaultwarden. There the data should be at the end of the day.Now I want to create a new container and I always get an error message. Where is my thinking error.

https://imgur.com/a/zYYZPcJ

Thank you.

**Comment by stefan0xC** [comment]
Not sure but unless you changed the `DATA_FOLDER` I think the mount point should be `/data` and not `/vault`.
